pkg/build/admission/admission.go

```diff
@@ -143,3 +154,11 @@ func (a *buildByStrategy) checkAccess(strategy buildapi.BuildStrategy, subjectAc
 func notAllowed(strategy buildapi.BuildStrategy, attr admission.Attributes) error {
 	return admission.NewForbidden(attr, fmt.Errorf("build strategy %s is not allowed", buildapi.StrategyType(strategy)))
 }
+
+func isValidUpdate(obj runtime.Object) (bool, error) {
+	accessor, err := meta.Accessor(obj)
+	if err != nil {
+		return false, err
+	}
+	return len(accessor.ResourceVersion()) > 0, nil
```

Comment:
Can't I do a force update without a `ResourceVersion`?  `Name` would have to be present.